book/listings/scalable_lists/1/integer_object/mod.rs

```diff
@@ -0,0 +1,16 @@
+mod imp;
+
+use glib::Object;
+use gtk::glib;
+
+// ANCHOR: integer_object
+glib::wrapper! {
+    pub struct IntegerObject(ObjectSubclass<imp::IntegerObject>);
+}
+
+impl From<i32> for IntegerObject {
+    fn from(number: i32) -> Self {
+        Object::new(&[("number", &number)]).unwrap()
```

Comment:
I would add a proper constructor to IntegerObject & use it in the From implementation.

Reply:
That's what I had before and @sdroege asked me to change it 😉

Reply:
That's because you had both a new & new_with_number or something like that :-)